sosw/worker.py

```diff
@@ -69,7 +69,12 @@ def __call__(self, event: Dict):
         # Mark the task as completed in DynamoDB if the event had task_id.
         try:
             if event.get('task_id'):
-                self.mark_task_as_completed(event['task_id'])
+                func_kwargs = {'task_id': event['task_id']}
+
+                if 'result' in event:
+                    func_kwargs.update({'result': event['result']})
+
+                self.mark_task_as_completed(**func_kwargs)
```

Comment:
Please pay more attention at your workflow and function signatures.
This behaviour would break the Worker __call__ again in most cases. >:(